mswindows/osgeo4w/package.sh

```diff
@@ -244,9 +240,11 @@ if [ -n "$PACKAGE_PATCH" ]; then
     unix2dos etc/postinstall/grass${PACKAGE_POSTFIX}.bat
     unix2dos etc/preremove/grass${PACKAGE_POSTFIX}.bat
 
-    # copy dependencies (TODO: to be reduced)
-    cp -uv $DLLS apps/grass/grass$POSTFIX/bin
-    cp -uv /mingw64/etc/fonts/fonts.conf apps/grass/grass$POSTFIX/etc
+    # copy dependencies
+    cp -uv $(/usr/bin/find apps/grass/grass$POSTFIX -iname "*.dll" -o -iname "*.exe" | PATH=$PWD/apps/grass/grass$POSTFIX/lib:$PWD/bin:/mingw64/bin:/usr/bin /usr/bin/xargs /usr/bin/ldd | /usr/bin/sed -ne 's#^.* => \(/mingw64/bin/.*\) (.*)$#\1#p' | /usr/bin/sort -u) apps/grass/grass$POSTFIX/bin
```

Comment:
if this is merged, DLLS can be removed

Reply:
I checked, and indeed, with these lines, the $DLLS variable isn't used anymore